content/documentation/glossary/index.adoc

```diff
@@ -0,0 +1,207 @@
+---
+title: "Glossary"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation"
+---
+
+= Kiali Overview
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Terminology Content
+:keywords: Kiali Documentation
+:icons: font
+:imagesdir: /images/documentation/terminology/
+
+== Application
+
+It's a logical group of <<Workload>> defined by the <<Label App>>.
+
+=== Application Name
+
+It's the name of the <<Application>> deployed in your environment. This name is provided by the <<App Label>> on the <<Workload>>
+
+
+== Deployment
+
+A deployment is a replication controller based on a user defined template called a <<Deployment>> configuration. Deployments are created manually or in response to triggered events.
+link:https://istio.io/help/glossary/#deployment[Istio Glossary]
+
+== Destination
+
+The remote upstream service Envoy is talking to on behalf of a source workload. There can be one or more service versions for a given service and Envoy chooses the version based on routing rules.
+
+link:https://istio.io/help/glossary/#destination[Istio Glossary]
+
+=== Destination Rule
+
+DestinationRule defines policies that apply to traffic intended for a service after routing has occurred. These rules specify configuration for load balancing, connection pool size from the sidecar, and outlier detection settings to detect and evict unhealthy hosts from the load balancing pool
+link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#DestinationRule[Istio Destination Rule]
+
+
+== Endpoints
+
+A communication endpoint is a type of communication network node. It is an interface exposed by a communicating party or by a communication channel
```

Comment:
I read 4 times the word communication here :)
I think the above statements should have some example/explanation how they relate to Istio/Kiali